.github/workflows/release.yml

```
@@ -63,3 +73,93 @@
+
+  homebrew:
+    needs: build
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/v')
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Get release assets and checksums
+        id: release_info
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2046:warning:7:9: Quote this to prevent word splitting

(shellcheck)